src/site/stages/build/process-cms-exports/schemas/input/node-health_services_listing.js

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable camelcase */
+
+module.exports = {
+  type: 'object',
+  properties: {
+    title: { $ref: 'GenericNestedString' },
+    created: { $ref: 'GenericNestedString' },
+    changed: { $ref: 'GenericNestedString' },
+    promote: { $ref: 'GenericNestedBoolean' },
+    sticky: { $ref: 'GenericNestedBoolean' },
+    default_langcode: { $ref: 'GenericNestedBoolean' },
+    revision_translation_affected: { $ref: 'GenericNestedBoolean' },
+    moderation_state: { $ref: 'GenericNestedString' },
+    metatag: { $ref: 'RawMetaTags' },
+    path: { $ref: 'RawPath' },
+    field_administration: {
```

Comment:
I used `field_administration: { $ref: 'EntityReferenceArray' },` which is the same thing I guess

Reply:
Sometimes the value will always be an array of exactly one `EntityReference`, so in those cases it is more specific to say that it is an array of on `EntityReference` instead of an `EntityReferenceArray`.